packages/shared/lib/core/wallet/interfaces/asset-balance.ts

```diff
@@ -0,0 +1,4 @@
+export interface IAssetBalance {
+    total: number
+    available?: number
+}
```

Comment:
The filename is missing the `.interface` file type specifier.

Reply:
Done